tox.ini

```diff
@@ -37,10 +37,13 @@ commands =
     vale sync --config="{toxinidir}/doc/.vale.ini"
     vale --config="{toxinidir}/doc/.vale.ini" "{toxinidir}/doc"
 
-[testenv:doc-{clean,links,html,pdf}]
+[testenv:doc-{clean,links,html,pdf,serve}]
 description = Checks documentation links and pages generates properly
+deps=
+    serve: sphinx-theme-builder[cli] @ https://github.com/pradyunsg/sphinx-theme-builder/archive/d9f620b1a73839728c95b596343595d3952ec8bf.zip
 allowlist_externals =
-    pdflatex
+    pdf: pdflatex
+    serve: stb
```

Comment:
This one is not required.

Reply:
<img width="1081" alt="Screenshot 2025-03-06 at 15 41 07" src="https://github.com/user-attachments/assets/59561b2c-1013-43c4-a057-017e2f82daeb" />
It was failing without adding stb

Reply:
The command is super useful! We are able now to hot-reload changes to the pages.

Reply:
Thank you @jorgepiloto for fixing it. For some reason, build is in infinite loop in my tox env when i build examples. i will retry with my new env.